.gitignore

```diff
@@ -5,3 +5,4 @@
 test/sauce.json
 test/node_modules/*
 test/coverage
+venv/
```

Comment:
This is not related to the issue you are closing, so it should be in a separate pull request.